data/roadmap/roadmap.yaml

```diff
@@ -48,133 +48,92 @@ categories:
     things:
       -
         status: Done
-        thing: "18F Phase I: Kickoff & Discovery"
+        thing: "18F Phase I: Discovery"
       -
         status: Done
-        thing: "18F Phase I: User Research"
+        thing: "18F Phase II: Search and Filter mapper alpha development"
       -
-        status: Done
-        thing: "18F Phase I: Synthesis & prioritization"
-      -
-        status: Done
-        thing: "18F Phase I: Roadmap & Low-fidelity prototyping"
+        status: on deck
+        thing: "Search and Filter mapper beta development"
       -
-        status: Done
-        thing: "18F Phase II: Planning"
+        status: on deck
+        thing: "Consolidated download beta development"
```

Comment:
Did we ever do an alpha development?

Reply:
We only did alpha code for Search and Filter, but we had alpha-level renderings (paper prototypes) that we used during that time for the download page.  You think this line is misleading?

Reply:
Not misleading, but maybe we could put that alpha phase is Done already

Reply:
That's fair.  Line added for that one to make it more clear.